cmake/AwsCFlags.cmake

```diff
@@ -24,6 +24,12 @@ function(aws_set_common_properties target)
     cmake_parse_arguments(SET_PROPERTIES "${options}" "" "" ${ARGN})
 
     if(MSVC)
+        # Remove other /W flags
+        if(CMAKE_C_FLAGS MATCHES "/W3")
```

Comment:
Do you want to search for /W{1-4} instead, just in case someone's gone mad with power?

Reply:
Oh sure